resources/views/docs/_structures/chat_channel.md

```diff
@@ -39,6 +40,7 @@ Optional attributes:
 Field                   | Type                                             | Description
 ----------------------- | ------------------------------------------------ | ------------------
 current_user_attributes | [CurrentUserAttributes](#currentuserattributes)? | only present on some responses
+first_message_id        | number?                                          |
```

Comment:
I don't think this exists anymore

Reply:
Doesn't seem returned on testing (though I don't have announce usergroup access to test that, in case it differs). It's been 22 months since I added support for the endpoint which uses this, so could definitely have been removed since then. 

I've removed it.